packages/data/package.json

```diff
@@ -23,6 +23,7 @@
 	"dependencies": {
 		"@babel/runtime": "^7.0.0",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/deprecated": "file:../deprecated",
```

Comment:
This means we should probably add `wp-deprecated` as a dependency of the `wp-data` script in `client-assets.php`

Reply:
Added in d0c6dba